src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$width-cell: 100px;
```

Comment:
Consider using a more descriptive variable name for the width of the calendar cell, such as $calendar-cell-width.